apps/betterangels-backend/accounts/migrations/0027_hmisprofile_hmisprofile_unique_hmis_id_agency.py

```diff
@@ -10,9 +10,8 @@ def add_hmis_profile(apps, schema_editor):
     ClientProfile = apps.get_model("accounts", "ClientProfile")
     HmisProfile = apps.get_model("accounts", "HmisProfile")
 
-    client_profiles = ClientProfile.objects.all()
-
-    for client_profile in client_profiles:
+    client_profiles_with_hmis_id = ClientProfile.objects.filter(hmis_id__isnull=False)
```

Comment:
non-blocker
```suggestion
    client_profiles_with_hmis_id = ClientProfile.objects.exclude(hmis_id=None)
```
do you happen to know if hmis_id could be `""` (empty string)? if so, might want to filter those out too

Reply:
fortunately, `ClientProfile.hmis_id` has a unique constraint, so there's only one "". Still a good idea; updated